README.md

```diff
@@ -47,9 +47,29 @@ At first glance these appear similar, but there is a distinction: the default sa
 
 The idea is to make it clear to other developers who might work on your project what dependencies are required for it to work and what are only needed for the dev environment.
 
+### npm scripts
+
+You may have noticed that npm created a section called "scripts" in your `package.json`. These are handy shortcuts you can create to run dependencies you install. Think of them as per project command line aliases.
+
+npm installs executable packages into `node_modules/.bin`, which means you can run them via the command line like this (for example) `node_modules/.bin/cowsay hello`. npm scripts automatically include this directory, so you can instead add a test script like this:
+
+```json
+{
+  "scripts": {
+    "hello": "cowsay hello"
+  }
+}
+```
+
+and run it with `npm run test`.
+
```

Comment:
This might be a good place to distinguish between scripts with names that npm recognise and attach special meaning to (`install`, `test`), and scripts with custom names. 
Worth mentioning the first is run using `npm <special-name>`, e.g `npm install`, and the latter require `npm run <custom name>`, e.g `npm run say-hello`.

Reply:
Technically leaving out the `run` is just a shorthand: `npm run test` will work fine. Do we want to mention the option or just keep it consistent for now and let them find out the shorthand later>

Reply:
I wasn't aware we could leave it in. I think you're right, consistency this early is better 👍